code/game/machinery/vendors/contraband_vendors.dm

```diff
@@ -0,0 +1,105 @@
+///Syndicate/Contraband Vendors
+
+/obj/machinery/economy/vending/wallmed/syndicate
+	name = "\improper SyndiMed Plus"
+	desc = "<b>EVIL</b> wall-mounted Medical Equipment dispenser."
+	icon_state = "syndimed"
+	icon_deny = "syndimed_deny"
+	ads_list = list("Go end some lives!","The best stuff for your ship.","Only the finest tools.","Natural chemicals!","This stuff saves lives.","Don't you want some?","Ping!")
+	req_access_txt = "150"
+	products = list(/obj/item/stack/medical/bruise_pack = 2,/obj/item/stack/medical/ointment = 2,/obj/item/reagent_containers/hypospray/autoinjector = 4,/obj/item/healthanalyzer = 1)
+	contraband = list(/obj/item/reagent_containers/syringe/charcoal = 4,/obj/item/reagent_containers/syringe/antiviral = 4,/obj/item/reagent_containers/food/pill/tox = 1)
+
+/obj/machinery/economy/vending/syndicigs
+	name = "\improper Suspicious Cigarette Machine"
+	desc = "Smoke 'em if you've got 'em."
+	slogan_list = list("Space cigs taste good like a cigarette should.","I'd rather toolbox than switch.","Smoke!","Don't believe the reports - smoke today!")
+	ads_list = list("Probably not bad for you!","Don't believe the scientists!","It's good for you!","Don't quit, buy more!","Smoke!","Nicotine heaven.","Best cigarettes since 2150.","Award-winning cigs.")
```

Comment:
```suggestion
	slogan_list = list("Space cigs taste good like a cigarette should.", "I'd rather toolbox than switch.", "Smoke!", "Don't believe the reports - smoke today!")
	ads_list = list("Probably not bad for you!", "Don't believe the scientists!", "It's good for you!", "Don't quit, buy more!", "Smoke!", "Nicotine heaven.", "Best cigarettes since 2150.", "Award-winning cigs.")
```